CalendarApp/BookingRoom/Models/CalendarConnection.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Google.Apis.Auth.OAuth2;
+using Google.Apis.Calendar.v3;
+using Google.Apis.Services;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography.X509Certificates;
+using System.Web;
+
+namespace BookingRoom.Models
+{
+    /// <summary>
+    /// Class for creating Google Caledar Service
+    /// </summary>
+    public class CalendarConnection
+    {
+        private string serviceEmail;
+        private X509Certificate2 certificate;
+        private ServiceAccountCredential credential;
+        private BaseClientService.Initializer initializer;
+        private CalendarService service;
+        /// <summary>
+        /// Constructor for connecting to Gmail Account
+        /// </summary>
+        public CalendarConnection()
```

Comment:
How are u going to test this class?
1. Inject dependencies
2. Remove hardcoded values